procrastinate/sql/migrations/00.19.00_02_null_locks_excluded.sql

```diff
@@ -0,0 +1,36 @@
+DROP FUNCTION IF EXISTS procrastinate_fetch_job(target_queue_names character varying[]);
```

Comment:
I think this will work with our deprecation policy but another brain validating that would be great :D

Reply:
Yes, it will work. The new SQL function has the same arguments signature as the old SQL function, so it is compatible with the Python code of previous Procrastinate versions.